### PR TITLE
Iss16 user view

### PR DESCRIPTION
**Pull request by LPMendoza**
## Description
The user view was developed integrating  the API developed.
Now the user can edit his profile information.

## How to test it
- Download the project
- Run the project
- Register in the app
- Enter to your profile
- Edit the information

## Screenshots
image
image
image
image

### CODE REVIEW

File: app/assets/stylesheets/sessions.scss [hunk 1]
```diff
@@ -1,4 +1,9 @@
+html {
+    height: 100%;
+}
 body {
-    background-image: url("back-circle.svg");
+    background-image: image-url("back-circle-gradient.svg");
     background-size: cover;
+    background-attachment: fixed;
+    height: 100%;
```
Comment (MiguelOcegueraM) [inline app/assets/stylesheets/sessions.scss:8]: nice one

File: app/controllers/profile_controller.rb [hunk 2]
```diff
@@ -2,39 +2,7 @@ class ProfileController < ApplicationController
...
   before_action :require_login
 
-  def show
-    @user = User.select(:id,:email,:name,:description).where(id: current_user.id)[0]
-    user = User.find(current_user.id)
-    render json: {
-      :user => @user,
-      :picture_link => url_for(user.profile_picture),
-    }
+  def home
```
Comment (MiguelOcegueraM) [inline app/controllers/profile_controller.rb:5]: ?????

**Review by MiguelOcegueraM — APPROVED** [review]
nice code, too clean

**Review by OchoaHub — APPROVED** [review]
Nice code

**Pull request merged**